docs/docs/smart-contracts/upgradeable.md

```diff
@@ -0,0 +1,693 @@
+---
+sidebar_position: 1
+title: Upgradeable contract
+---
+
+# Upgradeable contract
+
+## Overview
+
+Smart contracts are immutable by default, adding a layer of security and trust to the 
+contracts. But software quality depends on the ability to upgrade source code to 
+produce iterative releases. A certain degree of mutability is needed for bug fixing 
+and potential product improvements.
+
+Upgradeability allows for experimenting and deploying the product at the early stage, 
+always leaving the chance to fix vulnerabilities and progressively add features. 
+It is more actual right now while ink! and contract-pallet under active development. 
+Upgradeable contracts are not a Bug if they are developed consciously with 
```

Comment:
```suggestion
Upgradeable contracts are not a bug if they are developed consciously with 
```

Reply:
I tried to highlight it with capital letter=)